src/contributors/DavidThapaMagar.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "<David>"
+username: "<David-Thapa>"
+image: "<DavidThapa>"
```

Comment:
You dont have to include those `<>` brackets just provide information inside quotes.

Reply:
I have changed the format